### DISCUSSION TITLE
EPSG pour données Lidar (LAS/LAZ)

### DISCUSSION THREAD

**Discussion by pcardinal** [iTowns/itowns#2026] [Q&A]
Lors de la lecture d'un fichier LAZ, faut-il que le CRS du fichier soit identique à celui du viewer ?
Exemple :    var view = new itowns.View('EPSG:3946', viewerDiv);
Est-ce que le CRS de mon fichier LAZ doit aussi être EPS:3946 ou est-ce que les transformations de systèmes se font automatiquement si les deux CRS sont différents?

**Comment by jailln** [comment]
Oui il faut que le fichier LAS soit dans le même système de coordonnées que la vue
